lua/telescope/mappings.lua

```diff
@@ -30,7 +30,7 @@ mappings.default_mappings = config.values.default_mappings or {
 
     n = {
       ["<esc>"] = actions.close,
-      ["<CR>"] = actions.goto_file_selection_edit + actions.center,
+      ["<CR>"] = actions.select + actions.center,
       ["<C-x>"] = actions.goto_file_selection_split,
```

Comment:
How about those ? will they stay the same?

Reply:
If I understand what you're asking, I think the idea is that everything internally changes to the new naming scheme (split, vsplit, hsplit, etc.) and we leave aliases with the old names so that we don't break compatibility.